tests/integration-tests/tests/update/test_update.py

```diff
@@ -1178,7 +1178,7 @@ def _test_shared_storage_rollback(
         str(problematic_cluster_config),
         force_update="true",
         raise_on_error=False,
-        suppress_validators="type:SharedEbsVolumeIdValidator",
+        suppress_validators="ALL",
```

Comment:
In the future, lets improve the integration test code to support suppress two validators
```
           validator = kwargs.get("suppress_validators")
            if validator:
                validators_list.append(validator)
            kwargs["suppress_validators"] = validators_list
```